freqtrade/pairlist/pairlistmanager.py

```diff
@@ -110,6 +110,22 @@ def _prepare_whitelist(self, pairlist: List[str], tickers) -> List[str]:
 
         return pairlist
 
+    def verify_blacklist(self, pairlist: List[str], logmethod) -> List[str]:
```

Comment:
I gues the correct type for logmethod would be `Callable[..., Any]`

note: this is just an informative comment